Jenkinsfile

```diff
@@ -7,6 +7,10 @@ pipeline {
     stage ('Test') {
       steps {
         script { checkout scm }
+        sh '''
+            echo '[testenv]' >> tox.ini
+            echo 'basepython = python3.9' >> tox.ini
+        '''
```

Comment:
Don't add this kind of black magic without comment
Moreover, we already remove all this hackish snippet from other roles. Why did you want to re-add it?
We should add a dedicated worker instead of relying of the jenkins version